api-docs/getting-started/install-configure.rst

```diff
@@ -511,21 +511,9 @@ most recent versions of Ubuntu or Debian.
 
        deb <repo-name> cloudmonitoring main
 
-   ``repo-name`` is one of the following available packages:
+   ``repo-name`` is replaced with the following link with the appropriate package inserted:
 
-   -  http://stable.packages.cloudmonitoring.rackspace.com/ubuntu-10.04-x86\_64
-
-   -  http://stable.packages.cloudmonitoring.rackspace.com/ubuntu-12.04-x86\_64
-
-   -  http://stable.packages.cloudmonitoring.rackspace.com/ubuntu-14.04-x86\_64
-
-   -  http://stable.packages.cloudmonitoring.rackspace.com/ubuntu-14.10-x86\_64
-
-   -  http://stable.packages.cloudmonitoring.rackspace.com/ubuntu-15.04-x86\_64
-
-   -  http://stable.packages.cloudmonitoring.rackspace.com/debian-squeeze-x86\_64
```

Comment:
Unfortunately Debian uses the codename rather than the numerical ID like Ubuntu, so please also include a separate "debian" link that says "[Code name, such as squeeze]".